### DISCUSSION TITLE
在官方示例运行时，也出现 failed with code 1。

### DISCUSSION THREAD

**Discussion by WillWillWong** [open-compass/opencompass#959] [Q&A]
现在我配置第三方模型，进行评测。依旧存在03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_17] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_18] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_
19] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_
20] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_
21] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_
22] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_
23] failed with code 1
03/12 10:47:35 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/runners/base.py - summarize - 63 - OpenICLInfer[opt125m/math_24] failed with code 1
03/12 10:47:35 - OpenCompass - INFO - Partitioned into 1 tasks.

**Comment by bittersweet1999** [comment]
Please offer your logs so we can find the exact errors

**Reply by WillWillWong** [reply]
20240312_104855
tabulate format
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
dataset    version    metric    mode    sql_llama
---------  ---------  --------  ------  -----------
math       -          -         -       -
$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$

-------------------------------------------------------------------------------------------------------------------------------- THIS IS A DIVIDER --------------------------------------------------------------------------------------------------------------------------------

csv format
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
dataset,version,metric,mode,sql_llama
math,-,-,-,-
$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$

-------------------------------------------------------------------------------------------------------------------------------- THIS IS A DIVIDER --------------------------------------------------------------------------------------------------------------------------------

raw format
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-------------------------------
Model: sql_llama
$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$

**Comment by bittersweet1999** [comment]
![图片](https://github.com/open-compass/opencompass/assets/148421775/e62e958f-89a6-4358-bd90-fb0817c50faa)
Please provide this log, thank you

**Reply by WillWillWong** [reply]
Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
	Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.

**Reply by WillWillWong** [reply]
03/12 10:49:54 - OpenCompass - ERROR - /home/user/Project/opencompass/opencompass/tasks/openicl_eval.py - _score - 238 - Task [sql_llama/math]: No predictions found.
03/12 10:49:54 - OpenCompass - INFO - time elapsed: 2.33s

**Comment by bittersweet1999** [comment]
As observed in the logs, the model's inference has not been successfully initiated. If you are using OpenCompass on a Windows computer, you may encounter some compatibility issues. We recommend trying a Linux system instead.

**Reply by WillWillWong** [reply]
opencompass is running on linux, and I can run the model in other inference projects.Is it because I can't link to huggingface？

**Reply by bittersweet1999** [reply]
If so, try to do as follows:
```
export HF_EVALUATE_OFFLINE=1
export HF_DATASETS_OFFLINE=1
export TRANSFORMERS_OFFLINE=1
export TRANSFORMERS_CACHE=your model path
``` which means disable the downloading from hf, and change the model path

**Reply by WillWillWong** [reply]
$ cat outputs/default/20240312_111724/logs/infer/sql_llama/math_1.out
Error: mkl-service + Intel(R) MKL: MKL_THREADING_LAYER=INTEL is incompatible with libgomp.so.1 library.
        Try to import numpy first or set the threading layer accordingly. Set MKL_SERVICE_FORCE_INTEL to force it.

**Reply by WillWillWong** [reply]
Now that I have this problem, he still can't seem to use it

**Reply by bittersweet1999** [reply]
https://stackoverflow.com/questions/66932956/python-mkl-threading-layer-intel-is-incompatible-with-libgomp-so-1-library From google

**Reply by WillWillWong** [reply]
I tried to modify the version of numpy according to the link method, but the upgrade did not match the version of opencompass, so the version needs to be restored. pass export MKL_SERVICE_FORCE_INTEL=1 doesn't solve the problem.Thank you for your help.

**Reply by bittersweet1999** [reply]
how about this: https://blog.csdn.net/qq_40246742/article/details/117898715

**Reply by WillWillWong** [reply]
I was going through the other logs, and I found from llama import Llama.now,i don't can find this llama package,to solve this problem.

**Reply by WillWillWong** [reply]
Thanks for your help. Now I have replaced it with chatglm3, which can run now. But did not succeed in solving the above problems, but different environments to link hunggingface, which needs to be noted.
